### PR TITLE
Only have 'Add Tag' in manual tools, slight formatting

### PR DESCRIPTION
**Pull request by garyluu**
For issue ga4gh/dockstore#951

### CODE REVIEW

File: src/app/container/versions/versions.component.ts [hunk 1]
```diff
@@ -52,8 +52,16 @@ export class VersionsContainerComponent extends Versions implements OnInit {
...
+    if (this.tool && this.tool.mode === DockstoreTool.ModeEnum.MANUALIMAGEPATH) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
   setNoOrderCols(): Array<number> {
-    return [ 5, 6 ];
+    return [5, 6];
```
Comment (denis-yuen) [inline src/app/container/versions/versions.component.ts]: Not a focus of this PR, but this seems likely to break at some point as new columns are added and removed

Comment (garyluu) [reply src/app/container/versions/versions.component.ts]: Not actually sure what this does.  Workflow doesn't have it, just tools.

Comment (denis-yuen) [reply src/app/container/versions/versions.component.ts]: Does anything break if its removed? Worth checking out in a subsequent branch

Comment (garyluu) [reply src/app/container/versions/versions.component.ts]: I removed it at some point and at a glance did not notice any significant changes.  Will definitely take a closer look later.

Comment (garyluu) [reply src/app/container/versions/versions.component.ts]: setNoOrderCols makes the verified and action columns unsortable and there are different columns in the workflows vs tools.  So it does look necessary.

**Pull request merged**